### DISCUSSION TITLE
Peculiar wording in Using Databases with Vercel

### DISCUSSION THREAD

**Discussion by hirehamir** [vercel/community#3844] [Bug Report]
### Page to Investigate

https://vercel.com/guides/using-databases-with-vercel#i-want-my-data-to-be-available-globally.-which-storage-should-i-use

### Steps to Reproduce

> Using a traditional relational database in multiple regions is possible, but potentially costly or difficult to maintain. You might consider keeping your relational database regional and only **replication critical data to edge regions, otherwise keeping most workloads running in the single region with regionally located compute.**

The wording here seems off.

**Comment by amyegan** [comment]
Thanks for pointing that out. It's fixed now! 😄

**Reply by hirehamir** [reply]
You're welcome! I'm glad I could help.
